services-camel/src/main/java/edu/unc/lib/dl/services/camel/BinaryEnhancementProcessor.java

```diff
@@ -47,11 +53,28 @@ public void process(final Exchange exchange) throws Exception {
             Document msgBody = MessageUtil.getDocumentBody(in);
             Element body = msgBody.getRootElement();
 
-            String pidValue = body.getChild("pid", ATOM_NS).getTextTrim();
+            Element hasEnhancementPid = body.getChild("enhancements", ATOM_NS);
```

Comment:
has usually implies a boolean and isn't really what this is storing, maybe just enhancementEl or something like that?

Reply:
also, use the constant from JMSElementUtil for the name of this element